### ISSUE TITLE
0.9.16 marked as latest release in Github

### ISSUE THREAD

**Issue by aadamovich** [ratpack/ratpack#810]
Shouldn't it be 1.0.0 :)?

![image](https://cloud.githubusercontent.com/assets/2787794/10419370/5d7f5e08-707e-11e5-8d0e-682ad2fc666a.png)

**Comment by niwinz** [comment]
BTW, and would be awesome have a changelog file.

**Comment by erdi** [comment]
@niwinz each release page on ratpack.io has a list of closed issues and merged PRs, that's enough of a changelog, isn't it?

**Comment by johnrengelman** [comment]
I've updated the released manually with the same release notes from the milestones. We should integrate this into the release process though. Should be pretty straight forward.

**Comment by aadamovich** [comment]
Yep, GitHub seems to have an API for that: https://developer.github.com/v3/repos/releases/
